src/System.Security.Principal.Windows/tests/NTAccount.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using Xunit;
+
+namespace System.Security.Principal.Windows.Tests
+{
+    public class NTAccountTest
+    {
+        [Fact(Skip = "This test needs a machine in a domain but off line.")]
```

Comment:
Could you let it always run, and check that it either does not throw or throws Win32EXception? That would prove it does not throw ObjectDisposedException any more and occasionally it would likely get run on a machine in such a state. Eg. we often work on laptops with machines that are joined to the domain and do not have connectivity.